src/components/badges/Badge.jsx

```diff
@@ -1,49 +1,54 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import Text, {SIZE as TEXT_SIZE, COLOR as TEXT_COLOR, WEIGHT as TEXT_WEIGHT} from '../text/Text';
 
-export const SIZE = {
+export const BADGE_SIZE = {
   NORMAL: 'normal',
   SMALL: 'small',
   LARGE: 'large'
 };
 
-export const COLOR = {
-  NORMAL: {BADGE: 'light', TEXT: TEXT_COLOR.DEFAULT},
-  PEACH: {BADGE: 'peach', TEXT: TEXT_COLOR.LIGHT},
-  MUSTARD: {BADGE: 'mustard', TEXT: TEXT_COLOR.LIGHT},
-  MINT: {BADGE: 'mint', TEXT: TEXT_COLOR.LIGHT},
-  MINT_SECONDARY: {BADGE: 'mint-secondary', TEXT: TEXT_COLOR.LIGHT},
-  BLUE_SECONDARY: {BADGE: 'blue-secondary', TEXT: TEXT_COLOR.LIGHT},
-  BLUE: {BADGE: 'blue', TEXT: TEXT_COLOR.LIGHT},
-  GRAY_SECONDARY: {BADGE: 'gray-secondary', TEXT: TEXT_COLOR.LIGHT},
-  MINT_SECONDARY_LIGHT: {BADGE: 'mint-secondary-light', TEXT: TEXT_COLOR.MINT},
-  PEACH_SECONDARY_LIGHT: {BADGE: 'peach-secondary-light', TEXT: TEXT_COLOR.PEACH},
-  BLUE_SECONDARY_LIGHT: {BADGE: 'blue-secondary-light', TEXT: TEXT_COLOR.BLUE}
+export const BADGE_COLOR = {
```

Comment:
👍 I would keep the name `COLOR` though

Reply:
@maciekbb we prefix all constants - otherwise we always do 
```js
import {COLOR as BADGE_COLOR} from ...
```

Reply:
I changed it, as in most cases we have to make aliases (this allows us to have unique consts imported)